### ISSUE TITLE
Popover Storybook broken and window scroll resets on open

### ISSUE THREAD

**Issue by unscsprt** [priceline/design-system#631]
We noticed that the story for `Popover` doesn't display any content when its child is clicked. Clicking the child also resets the window's scroll position, which is a regression following recent fix https://github.com/pricelinelabs/design-system/pull/607, but I'm not sure if it's related to the aforementioned issue.

**Comment by qualitymanifest** [comment]
I'd like to give this a shot. I can't reproduce the scroll reset, but I think I've figured out why `Popover` isn't displaying any content. 

[The next commit](https://github.com/pricelinelabs/design-system/commit/fade1644002a450db623069b3bcd9c5d3d0d2c03#diff-e0ad6c0da699c344dec8823fea5e45ad) after #607 was merged upgraded `styled-components` from 3.4.10 to 4.4.0, and the console is full of warnings about `innerRef` being deprecated in `v4`. According to styled-components, [the solution](https://www.styled-components.com/docs/api#deprecated-innerref-prop) is to convert these to normal refs. This fixes the problem for me.

There is an additional warning in the console, however I went back to `checkout` the commit where #607 was merged and this warning is present there too.

I'll submit a PR in a bit

<details>
  <summary>The warning, for good measure</summary>

```
Warning: Function components cannot be given refs. Attempts to access this ref will fail. Did you mean to use React.forwardRef()?

Check the render method of `InnerReference`.
    in Button (created by InnerReference)
    in div (created by styled.div)
    in styled.div (created by InnerReference)
    in InnerReference (created by Context.Consumer)
    in Reference (created by Popover)
    in Manager (created by Popover)
    in Popover
    in div (created by Box)
    in Box
    in div (created by Box)
    in Box (created by Styled(Box))
    in Styled(Box) (created by Flex)
    in Flex
    in div (created by Box)
    in Box
    in div (created by styled.div)
    in styled.div (created by ThemeProvider)
    in ThemeProvider (created by ThemeProvider)
    in ThemeProvider
```
</details>

**Comment by BeniCheni** [comment]
#633
